core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -214,16 +216,15 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
 
     submissionFuture flatMap {
       case Some(submission) =>
-        if(submission.status == SubmissionStatuses.Aborting) {
+        val abortFuture = if(submission.status == SubmissionStatuses.Aborting) {
           for {
```

Comment:
So, the current code will call `abortQueuedWorkflows`, and `abortActiveWorkflows` in sequence (chained via callback).

However since these two methods are operating on mutually exclusive data (i.e. Queued workflows and Running workflows), I question whether they could be run in parallel instead. See:
```
        val abortFuture = if(submission.status == SubmissionStatuses.Aborting) {
          Future.sequence(Seq(abortQueuedWorkflows(submissionId), abortActiveWorkflows(submissionId))).mapTo[Unit]
        } else {
          Future.successful(())
        }
        abortFuture flatMap( _ => queryForWorkflowStatuses() )
```

Thoughts?